proposed/http-handlers/request-handlers-meta.md

```diff
@@ -363,6 +368,237 @@ executing that handler. If unable to route to a handler, it would instead
 execute the handler provided to the middleware. (This sort of mechanism can even
 be used in conjunction with middleware queues and stacks.)
 
+### 6.3 Example Interface Interactions
+
+The two interfaces, `RequestHandlerInterface` and `MiddlewareInterface`, were
+designed to work in conjunction with one another. While a
+`RequestHandlerInterface` can feasibly be used in isolation, it gains more power
```

Comment:
This sentence isn't necessary, imo. RequestHandler is perfectly viable *without* middleware, not the other way around.